### PR TITLE
Portfolio mobile version

### PR DESCRIPTION
**Pull request by Theophileaseh**
Branch created, HTML and CSS files updated

### CODE REVIEW

**Review by Whoistolu — COMMENTED** [review]
Hi @Theophileaseh, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that still needs the following requirement:

- [ ] Kindly set up linters correctly by following [this template](https://github.com/microverseinc/linters-config/tree/master/html-css)
- [ ] Please add more description to your Pull Request, stating the changes you have made to this milestone.
- [ ] Kindly ensure you closely follow the figma design for your template in every part of the project [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -114,6 +114,96 @@ <h3 class="skills">Skills</h3>
...
       </div>
-   </section> 
+   </section>
+
+   <section class="recent-work">
+      <h1 class="rw">MY RECENT WORK</h1>
+      <div class="divider"></div>
+      <div class="work1">
+         <div class="img1">
+            <img src="images/img.png" class="image" alt="software-portfolio">
```
Comment (Lordkaito) [inline index.html:124]: - [ ] Please make sure your images are being displayed on the screen. Right now this one is not being shown, it's showing the `alt` text instead. Same with all work images.

File: style.css [hunk 2]
```diff
@@ -198,3 +198,115 @@ section .circle2 {
 .portfolio .next .arrow-span {
   margin-left: 10px;
 }
+
+.recent-work {
+  padding: 30px;
+}
```
Comment (Lordkaito) [inline style.css:204]: - [ ] One of the requirements is to use CSS grid in your work section container, so please make sure to use `grid` to align your content inside.

File: index.html [hunk 3]
```diff
@@ -114,6 +114,96 @@ <h3 class="skills">Skills</h3>
...
+               <li class="works">html</li>
+               <li class="works">bootstrap</li>
+               <li class="works none">Ruby</li>
+            </ul>
+            <div class="button">
+               <a href="" class="main-button">See Project</a>
+            </div>
+         </div>
+      </div>
+      <div class="work1">
```
Comment (Lordkaito) [inline index.html:143]: - [ ] Another requirement is to use CSS flexbox to align the content inside the work card. So please make sure to implement the necessary changes to solve this issue.

File: index.html [hunk 4]
```diff
@@ -114,6 +114,96 @@ <h3 class="skills">Skills</h3>
...
+               <li class="works">bootstrap</li>
+               <li class="works none">Ruby</li>
+            </ul>
+            <div class="button">
+               <a href="" class="main-button">See Project</a>
+            </div>
+         </div>
+      </div>
+   </section>
+
```
Comment (Lordkaito) [inline index.html:207]: - [ ] For this milestone you need to implement the `About myself` section, which is missing right now, so please make sure to add this section as well.

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @Theophileaseh,

 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Hamzaoutdoors — APPROVED** [review]
Hi @Theophileaseh, 

## Congratulations! 🎉 Project Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀

To Highlight:

- [x] The Pull Request has a proper title and description. ✔️
- [x] You have added a descriptive Readme file. Good Job. ✔️
- [x] There is no linter error is present. ✔️
- [x] Project has met all the technical aspects you needed for this project! ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ]  Please make sure you add meaningful messages to your commits, `Add file` is very generic and if you find a bug you won't be able to trace down what commit caused it because all the commit messages are the same. Keep this in mind for your future projects as it is a good practice.
- [ ] Kindly regroup all your images assets in a folder `assets` that will make your project root more structured 

_Cheers and Happy coding and Keep rocking!_ 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**